test/validation/api/pktio/pktio.c

```diff
@@ -269,13 +269,15 @@ static void pktio_init_packet_eth_ipv4(odp_packet_t pkt, uint8_t proto)
 
 	/* Ethernet */
 	odp_packet_l2_offset_set(pkt, 0);
+	odp_packet_has_eth_set(pkt, true);
```

Comment:
This is not required by the API spec. Application is free to set it and not set it. Maybe validation test should not set it (do only minimum settings), so that implementations cannot depend on it.

Reply:
Ack.